api/protobuf-spec/dc.proto

```diff
@@ -21,6 +21,13 @@ message DcPutReq {
   bytes key = 1;
   bytes value = 2;
   float ttl_seconds = 3;
+  string etag = 4;
+}
+
+message DcDocument {
+  bytes value = 1;
+  string etag = 2;
```

Comment:
can u put these 2 values in a new object called `RecordMeta`. There will be more fields like ACL and many reasons to drop the meta before propagating the message

Reply:
Done.